### PR TITLE
Add workflow notifying team chat about pull requests

### PR DESCRIPTION
**Pull request by mdmathias**

### CODE REVIEW

File: .github/workflows/notification.yml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+name: GSI Chatroom PR Notification
+
+on:
+  pull_request:
+    types: [opened, reopened]
```
Comment (Alex-4-Git) [inline .github/workflows/notification.yml:5]: Does it make sense to use `review_requested` here? When a PR opens it is possible not ready yet.

File: .github/workflows/notification.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Pull Request Details
+      run: |
+        echo "Pull Request: ${{ github.event.pull_request.title }}"
+        echo "Author: ${{ github.event.pull_request.user.login }}"
+
+    - name: Google Chat Notification
+      run: |
+        curl --location --request POST '${{ secrets.WEBHOOK_URL }}' \
```
Comment (Alex-4-Git) [inline .github/workflows/notification.yml:22]: Where are the secrets stored?

**Review by Alex-4-Git — COMMENTED** [review]
Well done, Matt.

File: .github/workflows/notification.yml (hunk 2 above)
Comment (mdmathias) [reply .github/workflows/notification.yml:22]: They are stored on the [repo environment](https://docs.github.com/en/actions/security-guides/encrypted-secrets).

File: .github/workflows/notification.yml (hunk 1 above)
Comment (mdmathias) [reply .github/workflows/notification.yml]: Good question and point. I think it's nice to have a notification on `opened` as well. I've added a couple more that I think will be useful. Here is the [full list](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-object-34); take a look and let me know if you think others will be good to add.

Comment (mdmathias) [reply .github/workflows/notification.yml]: We call always modify this list if things get to noisy in the channel.

**Review by petea — APPROVED** [review]
Let's give it a shot :)

**Pull request merged**